core/capabilities/encoder_factory.go

```diff
@@ -0,0 +1,20 @@
+package capabilities
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/types"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+	"github.com/smartcontractkit/chainlink/v2/core/services/relay/evm"
+)
+
+func NewEncoder(name string, config *values.Map, lggr logger.Logger) (types.Encoder, error) {
+	switch name {
+	case "EVM":
+		return evm.NewEVMEncoder(config)
+	// TODO: add a "no-op" encoder for users who only want to use dynamic ones?
```

Comment:
Please make a ticket for TODOs and link to it.  Also, custom expresses it better than no-op.  IIUC, it'll be like a call to custom compute.